src/routes/App.jsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { Outlet } from 'react-router-dom';
+import Header from '../features/Header/Header';
+import { getMissions } from '../features/Mission/missionSlice';
+import { fetchRockets } from '../features/RocketList/rocketsSlice';
+
+const App = () => {
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRockets());
+    dispatch(getMissions());
```

Comment:
- [ ] According to the project requirements, Rockets is the default view, so its data would be fetched when the application starts (in `App` like you did 💯 ) However, the missions data should only be fetched (once) when a user navigates to the Missions section. Kindly fetch missions data when the missions component is rendered instead of fetching it when the app is started 👍